### DISCUSSION TITLE
Is there a way to fetch the very old tag (1.6)?

### DISCUSSION THREAD

**Discussion by belegnar** [Mailu/Mailu#3341] [User support] [answered]
Hi,

Current `rainloop` image is running but it's broken in the way where I can't save the image or push it somewhere (`docker push` or `docker save` results in a `no such file or directory` error). Please, help

**Comment by belegnar** [comment, marked as answer]
Could somebody put a tag on `de8e6f7`? It's `1.6` release

**Reply by nextgens** [reply]
There is a 1.6 branch; you can rebuild the images from that if you require them.

We haven't kept them when we had to move away from docker hub.
